assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm

```diff
@@ -200,6 +200,16 @@
 					$(this).prop('checked', newValue);
 				});
 			}
+
+			// SAK-29240
+			function deselectSelectAll( caller )
+			{
+				var checked = caller.checked;
+				if( !checked )
+				{
+					document.getElementById( "selectall" ).checked = checked;
```

Comment:
Sure, no problem. I generally try to avoid using jQuery for simple stuff. It's great for complex things as you can leverage the experience and maturity of the library, but I have experienced some annoyances and frustration with depending on jQuery when the syntax/APIs can change between versions.